docs/design/architecture.md

```diff
@@ -156,7 +383,7 @@ on par with MACVTAP.
 
```

Comment:
I tried to leave a comment on the picture but I noticed it is referencing QEMU and not the other hypervisors. Maybe just leave it as KVM if you do decide to change the pic?  Also, didn't kata-shim and kata-proxy go away? This looks like an older diagram that is v1.x specific.

Reply:
Yes, I'm not happy with the current diagrams. See #3245.